WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m

```diff
@@ -160,6 +160,9 @@ - (void)viewWillAppear:(BOOL)animated
         _trackedViewDisplay = true;
     }
 
+    // Manually deselect the selected row. This is required due to a bug in iOS7 / iOS8
+    [self.tableView deselectSelectedRowWithAnimation:true];
```

Comment:
Should this keep to the convention of using `YES` and `NO` for BOOL values in ObjC?  Is your Swift showing ;D

(also on line 160 above)